mm/slab.h

```diff
@@ -557,6 +558,8 @@ static inline size_t slab_ksize(const struct kmem_cache *s)
 static inline struct kmem_cache *slab_pre_alloc_hook(struct kmem_cache *s,
 						     gfp_t flags)
 {
+	struct kmem_cache *memcg_cache;
```

Comment:
This can just be declared in the scope where it's first used.

Reply:
Yes, I am sometimes too strict wrt declarations.